pkg/nativeimgutil/fuzz_test.go

```diff
@@ -0,0 +1,28 @@
+package nativeimgutil
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func FuzzConvertToRaw(f *testing.F) {
+	f.Fuzz(func(t *testing.T, imgData []byte, withBacking, specifySize bool) {
+		srcPath := filepath.Join(t.TempDir(), "src.img")
+		destPath := filepath.Join(t.TempDir(), "dest.img")
+		err := os.WriteFile(srcPath, imgData, 0o600)
+		if err != nil {
+			return
+		}
+		if specifySize {
+			f, err := os.Stat(srcPath)
+			if err != nil {
+				return
+			}
+			fileSize := f.Size()
```

Comment:
Just for my understanding, what is the reason to always pass the current filesize and not let the fuzzer modify it as well? Just to reduce the combinatorial space?

But it means it will never hit the error handling for a size too small, or trigger errors when the size is ridiculously large.

Also, won't `fileSize` always be the same as `len(imgData)`?